doc/source/notebooks/advanced/varying_noise.pct.py

```diff
@@ -78,12 +78,16 @@ def generate_data(N=80):
 
 # %%
 class HeteroskedasticGaussian(gpflow.likelihoods.Likelihood):
+    def __init__(self, **kwargs):
+        # this likelihood expects a single latent function F, and two columns in the data matrix Y:
+        super().__init__(latent_dim=1, observation_dim=2, **kwargs)
+
     def log_prob(self, F, Y):
         # log_prob is used by the quadrature fallback of variational_expectations and predict_density.
         # Because variational_expectations is implemented analytically below, this is not actually needed,
         # but is included for pedagogical purposes.
         # Note that currently relying on the quadrature would fail due to https://github.com/GPflow/GPflow/issues/966
-        Y, NoiseVar = Y[:, 0:1], Y[:, 1:2]
+        Y, NoiseVar = Y[:, 0], Y[:, 1]
```

Comment:
is this change because log_prob should now return a flat vector (not a [N, 1] data matrix)?